test/parallel/test-stdout-buffer-flush-on-exit.js

```diff
@@ -15,13 +15,11 @@ if (process.argv[2] === 'child') {
   process.exit();
 }
 
-[22, 21, 20, 19, 18, 17, 16, 16, 17, 18, 19, 20, 21, 22].forEach((exponent) => {
+[22, 16].forEach((exponent) => {
```

Comment:
>  It was making some test systems time out.

Yeah, I figured something like that was going on. Changing the test this way to avoid that problem sounds good to me (of course).